docs/docsource/operation/XDM_GUI/index.rst

```diff
@@ -0,0 +1,17 @@
+.. _The XDM UI:
+
+The XDM UI
+===========
```

Comment:
Does RST complain about an extra equal here? I can't remember how strict restructured is.

Reply:
Sphinx doesn't complain, but I'll fix it nonetheless.